packages/checkout-storefront/src/sections/PaymentSection/PaymentMethods.tsx

```diff
@@ -91,7 +23,9 @@ export const PaymentMethods: React.FC<PaymentMethodsProps> = ({
           className="shrink"
           value={paymentMethodId}
           selectedValue={selectedPaymentMethod || availablePaymentMethods[0]}
-          onChange={onSelect}
+          onChange={(event: ChangeEvent<HTMLInputElement>) =>
```

Comment:
```suggestion
          onChange={(event) =>
```

Reply:
We need it here, otherwise ts screms for no "value" inside this handler